rtd-requirements.txt

```diff
@@ -1,6 +1,6 @@
 certifi
 inflection==0.3.0
-requests==2.6.0
+requests==2.20.0
```

Comment:
What about 2.20.1 since that's the current `pypi` version? https://pypi.org/project/requests/